run-tests.js

```diff
@@ -11,6 +11,7 @@ import fs from 'fs'
 import ansi from 'ansicolor'
 import log from 'ololog'
 import ps from 'child_process'
+import { config } from './js/src/test/config.js'
```

Comment:
i think we should not import language-specific config here. we'd better have one universal json read (like we do have for skips now) and the `type`s and interfaces should live inside language-specific test files (`test.ts/py/php`, like at this moment we have defined classes there manually)

Reply:
rev: let's merge skips and configs in one json.